### PR TITLE
Change underlying MySQL type for a '.text' field to 'TEXT'. Issue #29…

### PR DESCRIPTION
**Pull request by bottlehall**
…0 refers.

### CODE REVIEW

File: Sources/MySQLKit/MySQLDialect.swift [hunk 1]
```diff
@@ -43,7 +43,7 @@ public struct MySQLDialect: SQLDialect {
     public func customDataType(for dataType: SQLDataType) -> SQLExpression? {
         switch dataType {
         case .text:
-            return SQLRaw("VARCHAR(255)")
+            return SQLRaw("TEXT")
```
Comment (tanner0101) [inline Sources/MySQLKit/MySQLDialect.swift:46]: The default here is `TEXT`: https://github.com/vapor/sql-kit/blob/master/Sources/SQLKit/Query/SQLDataType.swift#L29

So we can just remove this switch entirely and always return `nil`.

Comment (bottlehall) [reply Sources/MySQLKit/MySQLDialect.swift:46]: Won't the benchmark test still fail because it is creating a field with a default value?

Comment (tanner0101) [reply Sources/MySQLKit/MySQLDialect.swift:46]: Yeah, this note is unrelated. See my response on the #290 regarding that.

**Comment by bottlehall** [comment]
https://github.com/vapor/sql-kit/pull/118 has now completed tests. If approved, I think it should remove the errors in testing above and allow this PR to proceed.

**Comment by gwynne** [comment]
Closing due to age

**Comment by NathanFallet** [comment]
@gwynne The problem is still present and this PR should be merged to fix it!

**Pull request closed without merging**